checkov/dockerfile/checks/MaintainerExists.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.dockerfile.base_dockerfile_check import BaseDockerfileCheck
+
+
+class MaintainerExists(BaseDockerfileCheck):
+    def __init__(self):
+        name = "Ensure that LABEL maintainer is used instead of MAINTAINER (deprecated)"
+        id = "CKV_DOCKER_6"
+        supported_instructions = ["MAINTAINER"]
+        categories = [CheckCategories.CONVENTION]
+        super().__init__(name=name, id=id, categories=categories, supported_instructions=supported_instructions)
+
+    def scan_entity_conf(self, conf):
+        for instruction, content in conf.items():
+            if instruction == "MAINTAINER":
+                return CheckResult.FAILED, conf[instruction][0]
```

Comment:
I think the expression can be simplified into `content[0]`

Reply:
Yeah, you are right. Good point!